### ISSUE TITLE
loss during trainig

### ISSUE THREAD

**Issue by drazara** [rbgirshick/py-faster-rcnn#280]
I try train net/ and in training process i have this situations:
*********
I0801 19:06:38.232373 45998 solver.cpp:228] Iteration 2080, loss = 0.596326
I0801 19:06:38.232439 45998 solver.cpp:244]     Train net output #0: loss_bbox = 0.116154 (* 1 = 0.116154 loss)
I0801 19:06:38.232456 45998 solver.cpp:244]     Train net output #1: loss_cls = 0.151291 (* 1 = 0.151291 loss)
I0801 19:06:38.232471 45998 solver.cpp:244]     Train net output #2: rpn_cls_loss = 0.169351 (* 1 = 0.169351 loss)
I0801 19:06:38.232484 45998 solver.cpp:244]     Train net output #3: rpn_loss_bbox = 0.303964 (* 1 = 0.303964 loss)

I0801 19:08:26.177532 45998 solver.cpp:228] Iteration 2120, loss = 0.513034
I0801 19:08:26.177601 45998 solver.cpp:244]     Train net output #0: loss_bbox = 0.267161 (* 1 = 0.267161 loss)
I0801 19:08:26.177618 45998 solver.cpp:244]     Train net output #1: loss_cls = 0.225441 (* 1 = 0.225441 loss)
I0801 19:08:26.177630 45998 solver.cpp:244]     Train net output #2: rpn_cls_loss = 0.201166 (* 1 = 0.201166 loss)
I0801 19:08:26.177644 45998 solver.cpp:244]     Train net output #3: rpn_loss_bbox = 0.169628 (* 1 = 0.169628 loss)
*********
why my general loss is not sum of loss per layer?
I mean on Iteration 2120, loss = 0.513034, but loss_bbox + loss_cls + rpn_cls_loss + rpn_loss_bbox = 0.86.
It is significant for training quality?

**Comment by Oh233** [comment]
the summary loss is averaged over multiple iterations, while the train output is just the loss of current iteration.
